docs/src/installing-deis/installing-the-deis-platform.md

```diff
@@ -35,78 +35,156 @@ $ kubectl get pods --namespace=deis
 
 Once you see all of the pods ready, your Deis platform is running on a cluster!
 
-## Mapping a Default Domain
+## Routing Traffic
 
-Deis will route traffic from Kubernetes nodes to Deis components, but to
-do so it needs a domain. By default, Deis is configured to use the
-domain `example.com`. For basic Deis testing, simply add an entry to
-your host machine's `/etc/hosts` file.
+The Deis router component will use an incoming request's HTTP `Host`
+header to direct traffic to Deis components and to applications deployed
+via Deis. It is therefore important that the router be configured
+properly and that DNS and/or your local `/etc/hosts` file are as well.
+
+### Configuring the Platform Domain
+
+The Deis router requires a "platform domain" be set.  Deis components
+that respond to HTTP requests and applications deployed via Deis will
+all live on subdomains of the platform domain.  When installing Deis
+via helm, the platform domain is initially set to `example.com`.  This
```

Comment:
It's not a split infinitive if there's no verb after `to`. (Also, split infinitives are fine these days according to most style guides.)